test/e2e/serial/tests/workload_placement.go

```diff
@@ -328,8 +328,8 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			// there are now no more than 2 available CPUs under the targeted node and our test pod under the deployment is asking for 5 CPUs
 			// so in order to be certain that the pod will land on different node we need to request more than 7 CPUs in total
 			requiredRes = corev1.ResourceList{
-				// 6 here + 2 on the second container
-				corev1.ResourceCPU:    resource.MustParse("6"),
+				// 8 here + 2 on the second container
+				corev1.ResourceCPU:    resource.MustParse("8"),
```

Comment:
I'm not sure how this should help. Could you please elaborate about the math?

Reply:
I'm still not convinced by this specific change. I'm worried it can hide the real issue. Everything else LGTM.

Reply:
I'll remove this then and let's see if we can get some info from the scheduler logs as Talor suggested

Reply:
thanks!